frontend/public/style/_theme-dark.scss

```
@@ -0,0 +1,19 @@
+:root {
+  --co-global--palette--blue-600: var(--pf-global--palette--blue-600);
+  --co-global--palette--purple-600: var(--pf-global--palette--purple-600);
+  --co-global--palette--gold-700: var(--pf-global--palette--gold-700);
+  --co-global--palette--purple-700: var(--pf-global--palette--purple-700);
+
+  // dark
+  --co-global--dark--palette--blue-600: var(--pf-global--palette--blue-200);
+  --co-global--dark--palette--purple-600: var(--pf-global--palette--purple-200);
+  --co-global--dark--palette--gold-700: var(--pf-global--palette--gold-200);
+  --co-global--dark--palette--purple-700: var(--pf-global--palette--purple-200);
+}
+
+:root:where(.pf-theme-dark) {
+  --co-global--palette--blue-600: var(--co-global--dark--palette--blue-600);
+  --co-global--palette--purple-600: var(--co-global--dark--palette--purple-600);
+  --pf-global--palette--gold-700: var(--co-global--dark--palette--gold-700);
```

Related to above suggestion. 
Note you used `--pf-global--palette--gold-700` instead of `--co-global--palette--gold-700`
```suggestion
  --co-global--palette--orange-700: var(--co-global--dark--palette--orange-700);
```